### PR TITLE
Gsc/run options

### PR DESCRIPTION
**Pull request by dustinbleile**
Separation of working directory and workflow directory.

Allow Snakefile to be called from any folder.

### CODE REVIEW

File: scripts/merge_loh.R [hunk 1]
```diff
@@ -12,7 +12,27 @@ Options:
...
+       dplyr::pull(value)
+}
+this_file <- get_this_file()
+source(paste(dirname(this_file), "merge_mafs.R", sep="/"))
+
+# Find local script - method 2
+# Require install of new library funr
+# this_folder <- funr::get_script_path()
+# source(paste(this_folder, "merge_mafs.R", sep="/"))
+
```
Comment (dustinbleile) [inline scripts/merge_loh.R]: This section is the most questionable parts of the changes.

Comment (dustinbleile) [reply scripts/merge_loh.R]: This seemed like the nicest method of doing this import, but a new  R package was needed.

Comment (lculibrk) [reply scripts/merge_loh.R]: Instead of sourcing merge_mafs.R, I'm pretty sure we could do away with it and just add the functions to merge_loh.R?

Comment (dustinbleile) [reply scripts/merge_loh.R]: I like the idea of just adding the function to merge_loh.R
I'll try it.
I take it merge_mafs.R does not need to be a separate script as far as you can remember?

**Pull request merged**